salt/renderers/pass.py

```diff
@@ -45,6 +45,30 @@
 
         pass:
           pkg.installed
+
+Configuration options
+
+.. code-block:: yaml
+
+        # If the prefix is set, only the prefixed template variables are
+        # considered for fetching secrets from Pass. The default behavior
+        # is that *all* variables are considered.
+        # When the prefix matching is on, You may also consider
+        # setting the `renderer: jinja|yaml|pass` config option.
+        # recommended value: 'pass:'
+        pass_variable_prefix: 'pass:'
```

Comment:
can you make sure all of these new options are also added to salt/config/__init__.py with their defaults set.

Reply:
Done